cms/templates/js/content-group-details.underscore

```diff
@@ -41,17 +41,18 @@
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%= gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
       <ol class="usage group-configuration-usage">
         <% _.each(usage, function(unit) { %>
           <li class="usage-unit group-configuration-usage-unit">
-            <p><a href=<%= unit.url %> ><%= unit.label %></a></p>
+            <p><a href=<%= unit.url %> ><%- unit.label %></a></p>
           </li>
         <% }) %>
       </ol>
     <% } else { %>
         <p class="group-configuration-usage-text">
-        <%= outlineAnchorMessage %>
+          <!-- This contains an anchor link and therefore can't be escaped. -->
+          <%= outlineAnchorMessage %>
```

Comment:
@robrap Are you OK with Eric merging this as-is, and I finish it up as part of my work next Wednesday? I've already signed myself up for this file. :)